src/components/Game/Game.js

```diff
@@ -267,7 +264,7 @@ export default class Game extends Component {
       clues[dirToHide] = _.assign([], clues[dirToHide]).map((val) => val && '-');
     }
     const opponentGrid = this.opponentGame && this.opponentGame.grid;
-    const {screenWidth} = this.state;
+    const screenWidth = window.innerWidth - 1; // this is important for mobile to fit on screen
```

Comment:
This change isn't strictly necessary; but it avoids an initial render of the grid on mobile where all the grid cells are 1 px large. That would happen because screenWidth wasn't computed until `componentDidMount`, so the initial calculation ran as if `screenWidth = 0`.

Since nothing else ever modifies `screenWidth`, and it's a cheap calculation, there's no need to cache it.